.storybook/recipes/Filters/Filters.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import FiltersDrawer from '../../../src/components/FiltersDrawer';
+import {
+  FiltersPopover,
+  FiltersPopoverProps,
+} from '../../../src/components/FiltersPopover/FiltersPopover';
+
+type Props = {
+  /**
+   * Filters component to render.
+   */
+  as?: 'FiltersDrawer' | 'FiltersPopover';
+  /**
+   * Popover placement options relative to the filters trigger button.
+   */
+  placement?: FiltersPopoverProps['placement'];
+} & FiltersPopoverProps;
+
+/**
+ * Primary UI component for user interaction
```

Comment:
Could you update this component comment while you're in here?

Reply:
YUS will do

Reply:
Bump on this comment.